nixos/modules/services/networking/suricata/default.nix

```diff
@@ -22,7 +22,10 @@ let
     ;
 in
 {
-  meta.maintainers = with lib.maintainers; [ felbinger ];
+  meta = {
+    maintainers = with lib.maintainers; [ felbinger ];
+    buildDocsInSandbox = false;
```

Comment:
Only 43 hits for `git grep "buildDocsInSandbox = false"` in all of nixpkgs. I don't think that option should be set lightly.

I suspect things like `default = "${cfg.package}/etc/suricata/reference.config";` in `nixos/modules/services/networking/suricata/settings.nix` is why it fails. Try using `defaultText = ...` and make it build with the default settings?

Reply:
I agree, seams like these two default values we're responsible. I adjusted it as you suggested, thanks!